docs/api.rst

```diff
@@ -274,6 +274,13 @@ Check login status
    :statuscode 200: ok logged in
    :statuscode 401: no not logged in
 
+Get friends of a user
+~~~~~~~~~~~~~~~~~~~
+.. http:get:: /v1/account/friends
+
```

Comment:
Good job! Look at other routes that require parameters and try to add one.

```
:parameter email: email addr to find friends of
:type email: string
```

and maybe add an example reply:

```
**Example reply:**
.. sourcecode:: js
    ["turtle@rock.gov", "zebra@afri.ca"]
```